predicthq/version.py

```diff
@@ -1 +1 @@
-__version__ = "2.1.1"
+__version__ = "2.1.2"
```

Comment:
Should this not be a minor version bump at least as it is the addition of a new feature - not a bug fix or improvement?

Reply:
updated